README.md

```diff
@@ -42,11 +43,11 @@ With ice_cube, you can specify (in increasing order of precedence):
 * Recurrence Times - To specifically include in a schedule
 * Exception Times - To specifically exclude from a schedule
 
-Example: Specifying a recurrence with an exception time:
+Example: Specifying a recurrence with an exception time (ActiveRecord:
```

Comment:
Cool - and thanks so much for the PR!
If we can get that fixed I'm excited to merge!